crdt.go

```diff
@@ -473,8 +527,18 @@ func (store *Datastore) sendJobWorker() {
 		select {
 		case <-store.ctx.Done():
 			close(store.jobQueue)
+			// drain jobs
+			for j := range store.jobQueue {
+				j.session.Done()
+			}
```

Comment:
Hm. I'd generally expect the job queue worker to do this but either way works.

Reply:
yeah you're right